src/sentry/lang/native/minidump.py

```diff
@@ -2,10 +2,7 @@
 
 import logging
 
-import dateutil.parser as dp
-from msgpack import unpack, Unpacker, UnpackException, ExtraData
-
-from sentry.utils.safe import get_path, setdefault_path
+from sentry.utils.safe import get_path
```

Comment:
General comments on this file: It is essentially empty, apart from `is_minidump_event`. I'd propose to remove the file and move this into `sentry.lang.native.utils`.